pyalgostrategypool/inverse_ema_scalping_v2.py

```diff
@@ -0,0 +1,89 @@
+import talib
+
+from pyalgotrading.constants import *
+from pyalgotrading.strategy.strategy_base import StrategyBase
+
+
+class StrategyInverseEMAScalpingRegularOrder(StrategyBase):
+    name = 'Inverse EMA Scalping Regular Order Strategy v2'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.larger_time_period = self.strategy_parameters['LARGER_TIME_PERIOD']
+        self.smaller_time_period = self.strategy_parameters['SMALLER_TIME_PERIOD']
+
+        assert (0 < self.larger_time_period == int(self.larger_time_period)), f"Strategy parameter LARGER_TIME_PERIOD should be a positive integer. Received: {self.larger_time_period}"
+        assert (0 < self.smaller_time_period == int(self.smaller_time_period)), f"Strategy parameter SMALLER_TIME_PERIOD should be a positive integer. Received: {self.smaller_time_period}"
+
+        self.main_order_map = None
+
+    def initialize(self):
+        self.main_order_map = {}
+
+    def get_crossover_value(self, instrument):
+        # Get OHLC historical data for the instrument
```

Comment:
add a blank line before every comment, do this for all strategies

Reply:
removed all the comments 
done